core/server/config/index.js

```diff
@@ -49,7 +51,13 @@ _private.loadNconf = function loadNconf(options) {
      */
     nconf.set('env', env);
 
-    debug(nconf.get());
+    // Wrap this in a check, because else nconf.get() is executed unnecessarily
+    // To output this, use DEBUG=ghost:*,ghost-config
+    if (_debug.enabled('ghost-config')) {
+        debug(nconf.get());
```

Comment:
I've compared the boot time with and without `debug(nconf.get());` - no difference.